src/main/webapp/tutorials/facebook.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Seniors Connect</title>
+  <link rel="stylesheet" href="../styles/general.css">
+  <link rel="stylesheet" href="../styles/tutorial.css">
+  <link rel="import" href="../nav.html" />
+  <script src="scripts/login-script.js" defer></script>
+  <script src="https://unpkg.com/mustache@latest"></script>
+  <script type="module" src="../scripts/tutorial-scripts/tutorial-template.js"></script>
+  <script type="module" src="../scripts/tutorial-scripts/facebook-template.js"></script>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+</head>
+<body>
+  <div id="nav-placeholder"> </div>
+  <div id="facebook-placeholder"></div>
```

Comment:
Having separate `.html` files for each tutorial kind of defeats some of the gains from adding the template 😢  Instead of this, I wonder if we could use a url parameter to control which template gets loaded? So instead of `facebook.html` and others, you would have a single `tutorial.html`. When you direct to the URL, it would include a parameter like `/tutorial.html?tutorial=facebook` that would tell the javascript code which template file (like `facebook-template.js`) to invoke, which would load the correct data into the view.

(You don't necessarily have to do this now. May be worth a quick investigation - if it's easy to do, go for it. But don't fall down a rabbit hole trying to get it to work!)

Reply:
So I have been working on this the past day (with the help of Rae and Praneet) and we are running into some issues in terms of the order functions are called and loaded onto the page and I was unable to get it to work. I am planning on working on some styling and coming back to this later on.

Reply:
Got it - thanks for looking into it! Fine to leave for later.